be/src/exprs/jit/jit_engine.cpp

```diff
@@ -93,26 +85,30 @@ StatusOr<JITScalarFunction> JITEngine::compile_scalar_function(ExprContext* cont
 
     // TODO(Yueyang): optimize module name.
     auto expr_name = expr->debug_string();
-    JITScalarFunction compiled_function = nullptr;
-    compiled_function = (JITScalarFunction)instance->lookup_function_with_lock(expr_name);
 
+    auto compiled_function = (JITScalarFunction)instance->lookup_function_with_lock(expr_name, false);
     if (compiled_function != nullptr) {
         return compiled_function;
     }
 
     auto llvm_context = std::make_unique<llvm::LLVMContext>();
     auto module = std::make_unique<llvm::Module>(expr_name, *llvm_context);
     instance->setup_module(module.get());
+
     // Generate scalar function IR.
     RETURN_IF_ERROR(JITFunction::generate_scalar_function_ir(context, *module, expr));
-    if (llvm::verifyModule(*module)) {
-        return Status::JitCompileError("Failed to generate scalar function IR");
+    std::string error;
+    llvm::raw_string_ostream errs(error);
+    if (llvm::verifyModule(*module, &errs)) {
+        return Status::JitCompileError(
+                fmt::format("Failed to generate scalar function IR, verify failed: {}", errs.str()));
```

Comment:
Failed to generate scalar function IR, verify failed =>
Failed to generate scalar function IR, errors:

Reply:
Fixed.